docs/logging.md

```diff
@@ -0,0 +1,59 @@
+# Logging
+
+Using Bigflow you can easily add handlers to your loggers that will send all loggers calls to the Cloud Logging.
+In the CL you can filter logs per project id/logger name and workflow id. 
+
+## Getting started
+To use logging you have to install `bigflow` with `logger` module, `bigflow[logger]`.
+To create logger with the CL handler you have to use `configure_logging` method.
+
+```python
+from bigflow.logger import configure_logging
+project_id = 'some_project'
+logger_name ='some-logger'
```

Comment:
Why not using project name also as a logger name?

Reply:
It is more elastic. User can use project id but also may use another name.

Reply:
It makes sense. Probably we still can provide default value for logger name (so user *may* redefine it, but it is not obligated).